pandas/core/internals.py

```diff
@@ -5541,7 +5541,7 @@ def concatenate_join_units(join_units, concat_axis, copy):
     if len(to_concat) == 1:
         # Only one block, nothing to concatenate.
         concat_values = to_concat[0]
-        if copy and concat_values.base is not None:
+        if copy and getattr(concat_values, 'base', 1) is not None:
```

Comment:
Is this a hack, would like to have a better solution.

(related to discussion earlier this day in https://github.com/pandas-dev/pandas/pull/20721#discussion_r182716666 about deprecating Index.base)

Reply:
you should only check base if concat_values is an ndarray

Reply:
@jreback updated